### PR TITLE
lqty apr tooltip fixed

### PR DESCRIPTION
**Pull request by dvdese**
$LQTY_REWARDS value is multiplied by 0.5 twice when displaying LQTY APR tooltip

### CODE REVIEW

File: packages/dev-frontend/src/components/Stability/Yield.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export const Yield: React.FC = () => {
   const remainingLqtyOneYear = remainingStabilityPoolLQTYReward.mul(yearlyHalvingSchedule);
   const remainingLqtyInUSD = remainingLqtyOneYear.mul(lqtyPrice);
   const aprPercentage = remainingLqtyInUSD.div(lusdInStabilityPool).mul(100);
+  const remainingLqtyInUSDTooltip = remainingStabilityPoolLQTYReward.mul(lqtyPrice);
```
Comment (edmulraney) [inline packages/dev-frontend/src/components/Stability/Yield.tsx:44]: Hey :) the remaining LQTY is paid out over multiple years, so the stat your showing is not an APR (Annual). Instead you need to multiply by the amount which is distributed over the next year, which is 50% of the remaining rewards.

Comment (dvdese) [reply packages/dev-frontend/src/components/Stability/Yield.tsx:44]: Hi Edward, 
yes I think you are right, but the thing is that the total amount would be multiplied twice in the tooltip (although the final result is correct). If you look at the current numbers: _($532M * 50% / $1.25B) * 100 = 42.65%_  those 532M are already half (_~30.7 * 0.5 * 35_). Besides, _532 * 0.5 / 1250 * 100 = 21,28%_ . 
To clarify: I would only be changing the first number shown in the tooltip, not the result, nor the calculation itself 

tooltip_APR

File: packages/dev-frontend/src/components/Stability/Yield.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ export const Yield: React.FC = () => {
   const remainingLqtyOneYear = remainingStabilityPoolLQTYReward.mul(yearlyHalvingSchedule);
   const remainingLqtyInUSD = remainingLqtyOneYear.mul(lqtyPrice);
```
Comment (edmulraney) [inline packages/dev-frontend/src/components/Stability/Yield.tsx:42]:
```suggestion
  const remainingLqtyOneYearInUSD = remainingLqtyOneYear.mul(lqtyPrice);
```

File: packages/dev-frontend/src/components/Stability/Yield.tsx (hunk 1 above)
Comment (edmulraney) [reply packages/dev-frontend/src/components/Stability/Yield.tsx:44]: Ah! Gotcha. That makes sense. We should be showing total `$LQTY_REWARDS` remaining, not just this year's. Great spot! Thanks a lot @dvdese 

```suggestion
  const remainingLqtyInUSD = remainingStabilityPoolLQTYReward.mul(lqtyPrice);
```

File: packages/dev-frontend/src/components/Stability/Yield.tsx (hunk 2 above)
Comment (dvdese) [reply packages/dev-frontend/src/components/Stability/Yield.tsx]: Hi @edmulraney , I've updated the names as you propose. Glad to help (even a little bit ;-))

**Review by edmulraney — APPROVED** [review]
LGTM ⚡

Thank you very much for spotting this and fixing it! 💯

**Pull request merged**